README.md

```diff
@@ -10,9 +9,9 @@ https://matplotlib.org/mpl-sphinx-theme/.
 When creating a Matplotlib subproject you can include this theme by changing this
 line in your ``conf.py`` file
```

Comment:
Shouldn't all double backticks changed to single backticks?

Reply:
Thanks for catching - annoyingly that leaves enough changes that there isn't a nice overall diff, but looking at the diff of the two commits individually still seems to easily see what's changed.